src/studySetup.js

```diff
@@ -31,7 +31,7 @@ const baseStudySetup = {
     // default false. Actually send pings.
     send: true,
     // Marks pings with testing=true.  Set flag to `true` before final release
-    removeTestingFlag: true,
+    removeTestingFlag: false,
```

Comment:
Is this supposed to be false? Or are we ready for final release with this?

Reply:
whoops, didn't mean to change that